src/features/duplicates/components/ConfigureModal.tsx

```diff
@@ -11,12 +11,13 @@ import { FC, useState } from 'react';
 
 import FieldSettings from './FieldSettings';
 import messageIds from '../l10n/messageIds';
+import { PotentialDuplicate } from '../store';
+import PotentialDuplicatesLists from './PotentialDuplicatesLists';
 import useFieldSettings from '../hooks/useFieldSettings';
 import { useMessages } from 'core/i18n';
-import { ZetkinDuplicate } from '../store';
 
 interface ConfigureModalProps {
-  duplicate: ZetkinDuplicate;
+  duplicate: PotentialDuplicate;
```

Comment:
With the new name of the type, I think the name of this should change from `duplicate` to `potentialDuplicate`